packages/expo-clipboard/ios/ExpoClipboard.podspec

```diff
@@ -24,9 +24,9 @@ Pod::Spec.new do |s|
   }
 
   if !$ExpoUseSources&.include?(package['name']) && ENV['EXPO_USE_SOURCE'].to_i == 0 && File.exist?("#{s.name}.xcframework") && Gem::Version.new(Pod::VERSION) >= Gem::Version.new('1.10.0')
-    s.source_files = "#{s.name}/**/*.h"
+    s.source_files = "**/*.h"
```

Comment:
Do we still need to include h and m files?

Reply:
We don't, but I think it's a good idea to keep them here — there might be some cases where we would have to use Objective-C and in that case you don't have to care about the podspec. Also, it seems better to me to keep the podspecs in the same shape (especially with the module template).